app/styles/_core.less

```diff
@@ -105,6 +105,32 @@
     border-bottom: 0;
     padding-bottom: 0;
   }
+
+  .config-parameters-form {
```

Comment:
Nit:  alpha of this rule and the rules nested within

Reply:
Generally ignore the non-alpha characters as I think most folks don't think about 'em when they're scanning the code.  The overall goal is to make the Less more human-readable.  You would not believe the number of times I've seen rules declared multiple times or even the same declaration twice within the same rule.  Either way this rule should come first in this block.  :)